src/main/java/br/unb/cic/analysis/Main.java

```diff
@@ -167,8 +179,12 @@ protected void internalTransform(Body body, String phaseName, Map<String, String
                             case "dataflow"   : analysis = new ReachDefinitionAnalysis(body, definition); break;
                             case "tainted"    : analysis = new TaintedAnalysis(body, definition);
                             case "confluence" : analysis = new ConfluentAnalysis(body, definition); break;
-                            case "confluence-tainted": analysis = new ConfluentTaintedAnalysis(body, definition); break;
-                            case "overriding" : analysis = new OverridingAssignmentAnalysis(body, definition); break;
+                            case "confluence-tainted":
```

Comment:
@barbosamaatheus da próxima vez, se for fazer mudanças de indentação/formatação, isola isso em um commit específico. facilita a revisão

Reply:
Ok professor, ficarei mais atento a isso.